draft-thomson-webpush-protocol.xml

```diff
@@ -705,18 +843,19 @@ DATA         [stream 4] +END_STREAM
      <section title="Authorization">
        <t>
          This protocol does not define how a push service establishes whether a
-         user agent is permitted to create a subscription, or
-         whether push messages can be delivered to the user agent.  A push
+         user agent or application server is permitted to create a subscription,
```

Comment:
I don't think that we want to discuss application servers in quite the same context here.  You seem to think that the application server will have some pre-established relationship, I don't.  In fact, I think we've failed if there is that need.

I would rather discuss the implications for the application server subscriptions under denial of service considerations.